build/windows/installer/conf/azm-containers-parser.conf

```diff
@@ -0,0 +1,35 @@
+[PARSER]
+    Name         docker
+    Format       json
+    Time_Key     time
+    Time_Format  %Y-%m-%dT%H:%M:%S.%L
+    Time_Keep    On
+    # Command      |  Decoder | Field | Optional Action
+    # =============|==================|=================
+    # this was updated in 1.4.2 fluent-bit. Whenever upgrading fluent-bit, remember to keep this file in sync with the released parsers.conf file of the corresponding fluent version. 
+    #Decode_Field_As   escaped    log
+
+
+[PARSER]
+    # http://rubular.com/r/tjUt3Awgg4
+    Name cri
+    Format regex
+    Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
+    Time_Key    time
+    Time_Format %Y-%m-%dT%H:%M:%S.%L
+    Time_Keep    On
+
+
+[PARSER]
+    Name containerd-multiline
+    Format regex
+    Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log> *(\\u001b\[\d{1,2}m)* *(\[.*\])? *(((Mon|Tue|Wed|Thu|Fri|Sat|Sun|M|T|W|R|F|S|U|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday|Sunday) (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) \d{1,2} \d{1,2})|(\d{4}-\d{2}-\d{2})|(\d{2}:\d{2}:\d{2}((\.|,)\d{1,10})?)|(\d{8}T\d{6}((\.|,)\d{1,10})?)|(\d{10,17})|(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{0,9}Z)|(((\d{1,2} +(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) +\d{4,9})|((Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec) +\d{1,2},? +\d{4,9})) +((\d{1,2}:\d{1,2}:\d{1,2}((\.|,)\d{1,10})?)))).*)$
```

Comment:
is it possible to add the rubular linke like this CRI parser one  # http://rubular.com/r/tjUt3Awgg4 for these complex regexes?